docs/contributing/get-involved.md

```diff
@@ -2,4 +2,60 @@
+
+#### Beginner Resources:
+
+1. The “[Concepts](https://sourcecred.io/docs/concepts/cred)” Section of the SourceCred Website - this series of guides lays a foundation of the key terms and concepts that SourceCred is built upon like “Cred” and “Initiatives”.
+2. The [Code of Conduct ](https://sourcecred.io/docs/contributing/CODE_OF_CONDUCT) - a rough outlining of some of our community norms and values.
+3. The [Join a Community Call](https://discourse.sourcecred.io/t/how-to-use-discord-to-attend-a-community-call-proposal/725) rough draft - a protodocument that is still in the works but gives some basic information on how to join our weekly Community Call and how to join our Discord.
+4. The [Onboarding Machine](https://discourse.sourcecred.io/t/the-onboarding-machine-a-process-for-joining-sourcecred/726) Discourse Topic - Not so much a guide, but an explanation for the community of our thoughts on how to bring newcomers into our community effectively.
+5. The [How to Get an Ambassador](https://discourse.sourcecred.io/t/how-to-get-an-ambassador/727) How-To - a quick explanation of how to get person to person support as you enter the community and start contributing.
+
+# 3. Introduce Yourself
+
+We want to get to know you! Go to the SourceCred Discord and find the channel #intros. Post there letting us know that you’re poking into SourceCred and tell us a bit about yourself.
+
+You may want to share things like:
+
+* Your preferred name
+* What brought you to SourceCred
+* Other projects or activities you’re passionate about
+* Why you may want to work on SourceCred
+* Anything else you want to share!
+
+# 4. Come to a Community Call
+
+The SourceCred Community Call is a weekly voice call which we encourage anyone in the community to attend, but especially those folks who are new to SourceCred and want to know more about it.
+
+We encourage you to attend one and see how you like our community. We also would love it if you brought a question or two with you about things you’d like to understand better within SourceCred. That way, not only do you get a chance to learn but others on the call will have the chance as well!
+
+The Community Call is every Tuesday at 11am PT (double check your time zone) in the “General” voice channel on our Discord. For more step by step instructions on how to join the Discord voice channels for the Community Call, check out this [guide] (insert link once guide is finished, for now, see “Join a Community Call” linked above).
```

Comment:
Let's just leave this in there as a clear TODO, we'll fix it up shortly.

Reply:
We added a link to the community call calendar invite instead, and removed the TODO.